dspace-api/src/main/java/org/dspace/identifier/DOI.java

```diff
@@ -20,7 +20,7 @@
  * @author Pascal-Nicolas Becker
  */
 @Entity
-@Table(name = "Doi" )
+@Table(name = "doi" )
```

Comment:
While this doesn't look harmful, I find it odd that we'd need it lowercased here, when the Table name itself is uppercase: https://github.com/DSpace/DSpace/blob/master/dspace-api/src/main/resources/org/dspace/storage/rdbms/sqlmigration/postgres/V4.0__Upgrade_to_DSpace_4.x_schema.sql#L23

It makes me wonder if we should dig a bit deeper here. This error doesn't seem to happen in 6.x, so what changed in 7.x to result in it?  Are the Hibernate settings different? Do we have different versions of Hibernate that treat this differently?

Reply:
Maybe this is "expected".  As you noted, other Entities look to be lowercased, even when their table names are Uppercase.  Here's an example with the "webapp" entity:

* Entity is lowercase: https://github.com/DSpace/DSpace/blob/master/dspace-api/src/main/java/org/dspace/app/util/WebApp.java#L22
* Table is uppercase: https://github.com/DSpace/DSpace/blob/master/dspace-api/src/main/resources/org/dspace/storage/rdbms/sqlmigration/postgres/V4.0__Upgrade_to_DSpace_4.x_schema.sql#L43

So, that seems to imply this is expected behavior, and that your change is correct.

Reply:
Looking to that https://stackoverflow.com/a/20880247 I can guess that for some reason hibernate make validation queries quoting the table name, so forcing a case sensitive check. I really don't understand why this is not an issue with DSpace 6 but our tables were created without quotes so hey are allo lowercase and the hibernate annotation should reflect that. I'm going to merge as this issue prevent DSpace 7 to run